src/envoy/mixer/grpc_transport.cc

```diff
@@ -72,6 +82,18 @@ bool GrpcTransport::IsMixerServerConfigured(Upstream::ClusterManager& cm) {
   return cm.get(kMixerServerClusterName) != nullptr;
 }
 
+void GrpcTransport::onPreRequestCustomizeHeaders(Http::HeaderMap& headers) {
+  if (!headers_) return;
+
+  for (const auto& key : kHttpTraceHeaders) {
+    const HeaderEntry* entry = headers_->get(key);
```

Comment:
I manually tested it, it works.  I could not integration-test it since I have not figured out how to extract gRPC metadata in go,  It may require change mixer go code to extract gRPC metadata.